scripts/test_discord_webhooks.sh

```
@@ -0,0 +1,252 @@
+                {"name": "신뢰도", "value": "85%", "inline": true},
+                {"name": "주요 근거", "value": "1. 상승 추세 지속\n2. 거래량 증가\n3. RSI 과매도 탈출", "inline": false}
+            ]'
+            ;;
+        *)
+            print_error "Unknown test type: $test_type"
+            return 1
+            ;;
+    esac
+
+    # Send the webhook
+    local response=$(curl -s -w "\n%{http_code}" -X POST "$webhook_url" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)